src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -545,6 +545,23 @@ public void update(List<AbstractEndpoint> conferenceEndpoints, long bweBps)
                 synchronized (ssrcToSimulcastController)
                 {
                     ctrl = ssrcToSimulcastController.get(ssrc & 0xFFFF_FFFFL);
+
+                    if (ctrl != null)
+                    {
+                        if (trackBitrateAllocation.track != ctrl.getSource()) 
+                        {
+                            ssrcToSimulcastController.remove(ssrc & 0xFFFF_FFFFL);
+                            try {
+                              ctrl.close();
+                            }
+                            catch (Exception ignored)
+                            {
+            
+                            }
+                            ctrl = ssrcToSimulcastController.get(ssrc & 0xFFFF_FFFFL);
```

Comment:
This looks kind of like a retry of `ctrl = ssrcToSimulcastController.get(ssrc & 0xFFFF_FFFFL);`. What happens if `trackBitrateAllocation.track != ctrl.getSource()` after this second call?

Reply:
I believe this is effectively nulling out the ctrl variable. It might be clearer to set it to null